proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
```

Comment:
Juts a question - is this leading `_` conventional for syscalls?  I don't think I've seen it used in this way before.  That said - it seems fine in this context to distinguish the Win32 structs and API wrappers.